inst/include/component_data.hpp

```diff
@@ -82,6 +82,41 @@
 #define D_RF_halon2402          D_RF_PREFIX halon2402_COMPONENT_BASE
 #define D_RF_CH3Cl              D_RF_PREFIX CH3Cl_COMPONENT_BASE
 #define D_RF_CH3Br              D_RF_PREFIX CH3Br_COMPONENT_BASE
+
+// Adjusted halocarbon forcings
```

Comment:
Here too; "adjusted" or "relative"? To me, the latter is more precise and clearer.

Reply:
```suggestion
// Relative halocarbon forcings
```

Reply:
```suggestion
// Adjusted (i.e., relative) halocarbon forcings
```